sys/netinet/cc/cc_chd.c

```diff
@@ -161,7 +162,7 @@ should_backoff(int qdly, int maxqdly, struct chd *chd_data)
 {
 	unsigned long p, rand;
 
-	rand = random();
+	rand = prng32();
 
 	if (qdly < V_chd_qthresh) {
```

Comment:
I just wonder whether should_backoff() arguments here and below should really be signed, whether they makes sense and if so, whether it is handled right.

Reply:
I also am not sure why these were signed. I don't know hardly a thing about the code in question. Was just trying to take a stab at removing the deprecated `random()` call where possible.

Reply:
I don't know this code either, so it was just a thought.  What you should to though is use UINT32_MAX for RANDOM_MAX (or just inline it) and uint32_t for p.  rand variable I would just remove, inlining the prng32() call.

Reply:
Thanks! I did most of what you suggested, but I found that the rand variable was referenced twice, so it seemed necessary to keep it assigned. Is that alright?

Reply:
Oh, looks like RANDOM_MAX is still defined in the file. Should I remove it or reference it instead?

Reply:
> Is that alright?

Sure.  I've just missed the other use.  Sorry.

Either use RANDOM_MAX everywhere, or remove everywhere (in the two files for consistency).  It is used only in the same small function as prng32(), so it is a bit obvious, but a bit confusing being declared many screens away.